ext/uri/tests/rfc3986/modification/host_success_ipv6.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test Uri\Rfc3986\Uri component modification - host - IPv6 address
+--EXTENSIONS--
+uri
+--FILE--
+<?php
+
+$uri1 = Uri\Rfc3986\Uri::parse("https://example.com");
+$uri2 = $uri1->withHost("[2001:db8:3333:4444:5555:6666:7777:8888]");
```

Comment:
```suggestion
$uri2 = $uri1->withHost("[2001:0db8:3333:4444:5555:6666:7777:8888]");
```
Perhaps that to showcase whether or not IPv6 addressed will be normalized?

Reply:
P.S. I had to realize (again) that IPv6 addresses are not shortened during normalization. During normalization, only the casing of hexadecimal characters is corrected (they are either lowercased or uppercases).

But using an address that could be shortened is a good idea for sure.